test/verify/check-pages

```diff
@@ -396,6 +396,24 @@ OnCalendar=daily
 
         self.allow_restart_journal_messages()
 
+    def testShellPreload(self):
+        b = self.browser
+        m = self.machine
+
+        self.login_and_go()
+
+        # Check what's going on while playground/preloaded is still invisible
+        b.switch_to_top()
+        b.wait_present('iframe[name="cockpit1:localhost/playground/preloaded"][data-loaded]')
+        b.switch_to_frame("cockpit1:localhost/playground/preloaded")
+        b.wait_js_func('ph_text_is', "#host", m.execute("hostname").replace("\n", ""))
```

Comment:
This should also assert that `#release` is empty. I realize that this cannot  be done in a race free manner (the test may succeed even though the page would wrongly try to set it a bit later), but  it can't hurt.

Reply:
Yeah, that's why I didn't put it there...

Reply:
Done, after a little wait.